tool/util/go.go

```diff
@@ -0,0 +1,87 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"strings"
+
+	"github.com/open-telemetry/opentelemetry-go-compile-instrumentation/tool/ex"
+)
+
+// IsCompileCommand checks if the line is a compile command.
+func IsCompileCommand(line string) bool {
```

Comment:
Shall we add some unit test to functions like `IsCompileCommand` and `SplitCompileCmds` ?